logfire/_internal/exporters/console.py

```diff
@@ -182,7 +182,7 @@ def _details_parts(self, span: ReadableSpan, indent_str: str) -> TextParts:
         if not self._verbose or not span.attributes:
             return []
 
-        file_location: str = span.attributes.get('code.filepath')  # type: ignore
+        file_location = str(span.attributes.get('code.filepath'))
```

Comment:
```suggestion
        file_location = span.attributes.get('code.filepath')
        if file_location:
            file_location = str(file_location)
```
this should fix the failing test